Data Join can occur off device (#45)
patcg/docs-and-reports: design-dimensions/Dimensions-with-General-Agreement.md

The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should _at least_ include a time or interaction component to the budget scope. That is, as the user browses the web and interacts with sites more, we would expect that the private measurement system will allow more information about that user to be disclosed.

It is worth noting that, at least among existing differential privacy deployments, a time dimension in the privacy unit is very common. See https://desfontain.es/privacy/real-world-differential-privacy.html for a list of examples here.

## Location of Data Join

> The community group has reached general agreement that data join could potentially occur off device within a some type of server side architecture that achieves our desired security goals.

Reviewer: ```suggestion
The community group has reached general agreement that data join could potentially occur off device within a some type of server-side architecture that achieves our desired security goals.
```

...I couldn't resist.

Reviewer 2: ```suggestion
The community group has reached general agreement that data join could potentially occur off device within some type of server-side architecture that achieves our desired security goals.
```
One more nit on this line.

Reviewer: I might phrase this slightly differently though.

```suggestion
The community group has reached general agreement that data join could potentially occur off device within a some type of server-side architecture.
This is conditional on having adequate protections for any data that leaves a device, in line with our security and privacy goals.
```
